src/docs/clients/python/api.mdx

```diff
@@ -23,21 +23,21 @@ This gives you an overview of the public API that raven-python exposes.
 
 : This method is the low-level method for reporting events to Sentry. It captures and processes an event and pipes it via the configured transport to Sentry.
 
-  Example:
-
-  ```python
-  capture('raven.events.Message', message='foo', data={
-      'request': {
-          'url': '...',
-          'data': {},
-          'query_string': '...',
-          'method': 'POST',
-      },
-      'logger': 'logger.name',
-  }, extra={
-      'key': 'value',
-  })
-  ```
+Example:
+
+```python
+capture('raven.events.Message', message='foo', data={
+    'request': {
+        'url': '...',
+        'data': {},
+        'query_string': '...',
+        'method': 'POST',
+    },
+    'logger': 'logger.name',
+}, extra={
+    'key': 'value',
+})
+```
```

Comment:
we also want to keep these leading spaces since they cause the code block to be indented in the text

I think a similar thing is happening with bullets but haven't run across the example in this PR

Reply:
Let's confirm in the build preview (assuming it builds)

Reply:
nvm it also failed